.travis.yml

```diff
@@ -0,0 +1,13 @@
+sudo: false
+language: node_js
+node_js: node
+install:
+- travis_retry gem install s3_website -v 3.4.0
+- travis_retry pip install awscli --upgrade --user
+- travis_retry npm install
+before_script: npm run build
```

Comment:
As mentioned in a previous PR comment and based on [this post](https://stackoverflow.com/a/49216334), the build should be performed in the `script` step rather than the `before_script` step. I went back and changed the instructions in the S3 deploy document to reflect this recommendation.